elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AsyncProperties.java

```diff
@@ -39,4 +39,14 @@
      */
     private boolean defaultAsyncQueryDAO = true;
 
+    /**
+     * Whether or not the controller is enabled.
+     */
+    private boolean enabled = false;
+
+    /**
+     * The URL path prefix for the controller.
+     */
+    private String downloadPath = "/";
```

Comment:
We already have a way to configure controllers.  Should this be a subclass of the a controller?